asab/api/service.py

```diff
@@ -24,6 +21,42 @@ def __init__(self, app, service_name="asab.ApiService"):
 		super().__init__(app, service_name)
 		self.WebContainer = None
 		self.ZkContainer = None
+		self.AttentionRequired = []  # list of errors found.
+
+	def attention_required(self, error_id=None):
+
```

Comment:
```
if "_c" is not in att:
    att["_c"] = datetime.datetime.utcnow().isoformat() + 'Z'
```

So that the att info contains also the creation time - it will be handy.